### PR TITLE
[TE] frontend - ttbach/Fixed eslint errors

### PR DESCRIPTION
**Pull request by ttbach**

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js [hunk 1]
```diff
@@ -201,7 +201,6 @@ export default Component.extend({
     'entities',
     'aggregates',
     function () {
-      const { entities, aggregates } = this.getProperties('entities', 'aggregates'); // poll observer
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js]: Is this safe to remove this? I believe @apucher was intentionally triggering a cp recomputation here.

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js [hunk 2]
```diff
@@ -240,7 +239,6 @@ export default Component.extend({
...
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js]: same

**Comment by apucher** [comment]
yeah just looked into this too. I'll address this here https://github.com/linkedin/pinot/pull/2517

**Comment by ttbach** [comment]
@apucher Ok, then I'll close this PR and https://github.com/linkedin/pinot/pull/2514 then.

**Pull request closed without merging**